pkg/generated/openapi/zz_generated.openapi.go

```diff
@@ -9148,7 +9148,7 @@ var OpenAPIDefinitions *common.OpenAPIDefinitions = &common.OpenAPIDefinitions{
 	"v1.Node": {
 		Schema: spec.Schema{
 			SchemaProps: spec.SchemaProps{
-				Description: "Node is a worker node in Kubernetes, formerly known as minion. Each node will have a unique identifier in the cache (i.e. in etcd).",
```

Comment:
This is a bug in verification code (#33819) that somebody changed a type comment and didn't check in generated openapi file. This needed for me here because now checked-in specs reply on this file, and the new verify openapi check will fail if this file is not up to date.